dev-docs/bidders/silverpush.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: bidder
+title: Silverpush
+description: Prebid Silverpush Bidder Adaptor
+pbjs: true
+pbs: true
+biddercode: silverpush
+media_types: banner, video
+schain_supported: true
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
+floors_supported: true
+userIds: 
+prebid_member: false
+fpd_supported: false
+multiformat_supported: will-bid-on-one
+sidebarType: 1
+---
+
+### Registration
+The Example Bidding adapter requires setup before beginning. Please contact us at prebid@silverpush.co.
+
+
+### Bid Parameters
+
+#### Banner
+
+{: .table .table-bordered .table-striped }
+|       Name     |    Scope    |                 Description                |    Example    |     Type     |
+| -------------- | ----------- | ------------------------------------------ | ------------- | ------------ |
+| `publisherId` | required | Publisher id provided by silverpush | "123456" | String |
+| `bidFloor` | optional | Minimum price in USD. bidFloor applies to a specific unit. For example, use the following value to set a $1.50 floor: 1.50. <br/> | 1.50 | Number |
+
+
+#### mediaTypes.banner
+
+The following banner parameters are supported here so publishers may fully declare their banner inventory:
+
+{: .table .table-bordered .table-striped }
+|    Name   |    Scope    |                      Description                                  |  Example  |    Type   |
+| --------- | ------------| ----------------------------------------------------------------- | --------- | --------- |
+| sizes | required | Avalaible sizes supported for banner ad unit | [ [300, 250], [300, 600] ] | [[Integer, Integer], [Integer, Integer]] | 
+
+
+### AdUnit Format for Banner
+```javascript
+const adUnits = [{
+    code: 'div-1',
+    mediaTypes: {
+        banner: {
+            sizes: [ [300, 250], [300,600] ]
+        }
+    },
+    bids: [{
+      bidder: 'silverpush',
+      params: {
+          publisherId: "123456",
+          bidFloor: 1.2
+      }
+    }]
+}];
+```
+
+#### Video
+
+| Name | Scope | Description | Example | Type |
+| ---- | ----- | ----------- | ------- | ---- |
+| `publisherId` | required | Publisher id provided by silverpush | "123456" | String |
+| `bidFloor` | optional | Minimum price in USD. bidFloor applies to a specific unit. For example, use the following value to set a $1.50 floor: 1.50. <br/> | 1.50 | Number |
```

Comment:
The layout of the table is broken. I assume because of the html?
```suggestion
| Name | Scope | Description | Example | Type |
| ---- | ----- | ----------- | ------- | ---- |
| `publisherId` | required | Publisher id provided by silverpush | "123456" | String |
| `bidFloor` | optional | Minimum price in USD. bidFloor applies to a specific unit. For example, use the following value to set a $1.50 floor: 1.50. | 1.50 | Number |
```

Check: https://deploy-preview-4510--prebid-docs-preview.netlify.app/dev-docs/bidders/silverpush.html

Reply:
Missed table classes before the description, changes done. 😄

Reply:
Awesome 😎